motor/providers/awsec2ebs/types.go

```diff
@@ -53,16 +53,18 @@ type SnapshotId struct {
 	Account string
 }
 
-type VolumeId struct {
+type VolumeInfo struct {
 	Id          string
 	Region      string
 	Account     string
 	IsAvailable bool
+	Tags        map[string]string
 }
 
 func resourceTags(resourceType types.ResourceType, instanceId string) []types.TagSpecification {
 	return []types.TagSpecification{
-		{ResourceType: resourceType,
+		{
+			ResourceType: resourceType,
 			Tags: []types.Tag{
 				{Key: aws.String("Created By"), Value: aws.String("Mondoo")},
```

Comment:
Should we stick to the standard AWS tag "createdBy" https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/aws-tags.html?

Reply:
Changed.